ui/src/app/kvm/views/KVMDetails/RamResources/RamResources.tsx

```diff
@@ -70,66 +70,68 @@ const RamResources = ({
           size={96}
         />
       </div>
-      <table className="ram-resources__table">
-        <thead>
-          <tr>
-            <th></th>
-            <th className="u-align--right u-text--light">
-              <span className="u-nudge-left">Allocated</span>
-            </th>
-            <th className="u-align--right u-text--light">
-              <span className="u-nudge-left">Free</span>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td>General</td>
-            <td className="u-align--right">
-              {memoryWithUnit(general.allocated)}
-              <span className="u-nudge-right--small">
-                <i className="p-circle--link"></i>
-              </span>
-            </td>
-            <td className="u-align--right">
-              {memoryWithUnit(general.free)}
-              <span className="u-nudge-right--small">
-                <i className="p-circle--link-faded"></i>
-              </span>
-            </td>
-          </tr>
-          {totalHugepages > 0 && (
-            <tr data-test="hugepages-data">
-              <td>
-                Hugepage
-                {hugepages.pageSize && (
-                  <>
-                    <br />
-                    <strong
-                      className="p-text--x-small u-text--light"
-                      data-test="page-size"
-                    >
-                      {`(Size: ${memoryWithUnit(hugepages.pageSize)})`}
-                    </strong>
-                  </>
-                )}
-              </td>
+      <div className="ram-resources__table-container">
```

Comment:
Driveby fix for a bug in Firefox where the table cells would take up extra space. Wrapping the table in a div seems to fix it - tables don't seem to like directly interacting with flexbox.